README.md

```diff
@@ -12,63 +12,60 @@ This CDK library automatically configures ingestion of metrics, traces, and logs
 - Enabling the collection of traces and custom metrics from your Lambda functions.
 - Managing subscriptions from the Datadog Forwarder to your Lambda function log groups.
 
-## Installation
-
+## Npm Package Installation:
 ```
 yarn add --dev datadog-cdk-constructs
 # or
 npm install datadog-cdk-constructs --save-dev
 ```
 
+## PyPi Package Installation:
+```
+pip install datadog-cdk-constructs
+```
+
 ## Usage
 
 ### AWS CDK
 
-import datadog-cdk-constructs.
-
-**Typescript**
-
+Add this to your CDK stack:
 ```typescript
-import * as cdk from "@aws-cdk/core";
 import { Datadog } from "datadog-cdk-constructs";
 
-class CdkStack extends cdk.Stack {
-  constructor(scope: cdk.Construct, id: string, props?: cdk.StackProps) {
-    super(scope, id, props);
-    const datadog = new Datadog(this, "Datadog", {
-      nodeLayerVersion: <LAYER_VERSION>,
-      pythonLayerVersion: <LAYER_VERSION>,
-      addLayers: <BOOLEAN>,
-      forwarderARN: "<FORWARDER_ARN>",
-      flushMetricsToLogs: <BOOLEAN>,
-      site: "<SITE>",
-      apiKey: "{Datadog_API_Key}",
-      apiKMSKey: "{Encrypted_Datadog_API_Key}",
-      enableDDTracing: <BOOLEAN>,
-      injectLogContext: <BOOLEAN>
-    });
-    datadog.addLambdaFunctions([<LAMBDA_FUNCTIONS>])
-  }
-}
+const datadog = new Datadog(this, "Datadog", {
+  nodeLayerVersion: <LAYER_VERSION>,
+  pythonLayerVersion: <LAYER_VERSION>,
+  addLayers: <BOOLEAN>,
+  forwarderArn: "<FORWARDER_ARN>",
+  flushMetricsToLogs: <BOOLEAN>,
+  site: "<SITE>",
+  apiKey: "{Datadog_API_Key}",
+  apiKmsKey: "{Encrypted_Datadog_API_Key}",
+  enableDatadogTracing: <BOOLEAN>,
+  injectLogContext: <BOOLEAN>
+});
+datadog.addLambdaFunctions([<LAMBDA_FUNCTIONS>])
```

Comment:
I modified this because JSII was not producing a code snippet that was correct for the PyPi Module but now its just the construct code without the CDK stack code.

Reply:
Basically the constructor that JSII was generating would fail if someone copied and pasted it, I figure that we want the snippet to be as correct and with very little work for the user to get it up and running. This assumes the user knows how to add it to their CDK stack. That may be a bad assumption for beginners so I think I will link a CDK workshop.